marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/builtins/steps/mapping/map-multiple-entities/runMapping.sjs

```diff
@@ -0,0 +1,107 @@
+      )
+    );
+  }
+
+  try {
+    assertions = assertions.concat([
+      test.assertEqual(10, fn.number(instance1.Product.productId)),
+      test.assertEqual('New Balance FuelCell Echo V1 Sneaker', fn.string(instance1.Product.productName))
+    ]);
+  } catch (e) {
+    assertions.push(
+      test.assertFalse(
+        fn.true(),
+        `Error "${e.toString()}" encountered testing instance1 '${xdmp.describe(instance1, Sequence.from([]), Sequence.from([]))}'`
+      )
+    );
+  }
+
+  try {
+    assertions = assertions.concat([
+      test.assertEqual(30, fn.number(instance2.Product.productId)),
+      test.assertEqual('Crock-Pot 8 Quart Manual Slow Cooker with 16 Oz Little Dipper Food Warmer, Stainless', fn.string(instance2.Product.productName))
+    ]);
+  } catch (e) {
+    assertions.push(
+      test.assertFalse(
+        fn.true(),
+        `Error "${e.toString()}" encountered testing instance2 '${xdmp.describe(instance2, Sequence.from([]), Sequence.from([]))}'`
+      )
+    );
+  }
+  try {
+    assertions = assertions.concat([
+      test.assertEqual(40, fn.number(instance3.Product.productId)),
+      test.assertEqual('Cuisinart 8-Cup Stainless Steel Electric Kettle with Automatic Shut-off', fn.string(instance3.Product.productName))
+    ]);
+  } catch (e) {
+    assertions.push(
+      test.assertFalse(
+        fn.true(),
+        `Error "${e.toString()}" encountered testing instance3 '${xdmp.describe(instance3, Sequence.from([]), Sequence.from([]))}'`
+      )
+    );
+  }
+  try {
+    assertions = assertions.concat([
+      test.assertEqual(2002, fn.number(instance4.Order.orderId)),
+      test.assertEqual(303, fn.number(instance4.Order.deliveredTo)),
+      test.assertEqual(1, fn.number(instance4.Order.lineItems[0].LineItem.quantity)),
+      test.assertEqual(10, fn.number(instance4.Order.lineItems[0].LineItem.orderIncludes))
+    ]);
+  } catch (e) {
+    assertions.push(
+      test.assertFalse(
+        fn.true(),
+        `Error "${e.toString()}" encountered testing instance4 '${xdmp.describe(instance4, Sequence.from([]), Sequence.from([]))}'`
+      )
+    );
+  }
+  try {
+    assertions = assertions.concat([
+      test.assertEqual(2012, fn.number(instance5.Order.orderId)),
+      test.assertEqual(404, fn.number(instance5.Order.deliveredTo)),
+      test.assertEqual(1, fn.number(instance5.Order.lineItems[0].LineItem.quantity)),
+      test.assertEqual(30, fn.number(instance5.Order.lineItems[0].LineItem.orderIncludes)),
+      test.assertEqual(2, fn.number(instance5.Order.lineItems[1].LineItem.quantity)),
+      test.assertEqual(40, fn.number(instance5.Order.lineItems[1].LineItem.orderIncludes))
+    ]);
+  } catch (e) {
+    assertions.push(
+      test.assertFalse(
+        fn.true(),
+        `Error "${e.toString()}" encountered testing instance5 '${xdmp.describe(instance5, Sequence.from([]), Sequence.from([]))}'`
+      )
+    );
+  }
+}
+runMapping();
```

Comment:
Since there's only the one test (which is a good practice), let's remove the function - just have this all in the "main" part of the module, no need for a function.